src/Microsoft.SqlTools.ServiceLayer/Workspace/WorkspaceService.cs

```diff
@@ -329,6 +329,7 @@ public void RegisterTextDocOpenCallback(TextDocOpenCallback task)
             {
                 Logger.Write(TraceEventType.Verbose, "HandleDidChangeConfigurationNotification");
 
+                this.CurrentSettings = configChangeParams.Settings;
```

Comment:
looks like you are fixing a bug here, how does other services work without this value being up to date?

Reply:
Most services that are using configs have registered a handler and are just using that.